services/frontend-service/src/ui/utils/Links.test.tsx

```diff
@@ -189,9 +191,9 @@ describe('ReleaseVersionLink', () => {
     ];
     describe.each(cases)('RendersProperly', (testcase) => {
         const getNode = () => (
-            <DisplayLink
+            <DisplayManifestLink
```

Comment:
Isn't this a bit too much too link it again?

Reply:
In the test? Not sure what you mean by that @hannesg 

I will add one more test to also show that the `DisplayManifestLink` renders empty if not all input is given.

Reply:
I added more tests